package.json

```diff
@@ -131,17 +102,45 @@
     "eslint-plugin-jsdoc": "^48.2.1",
     "eslint-plugin-n": "^16.6.2",
     "eslint-plugin-promise": "^6.1.1",
+    "fast-glob": "^3.3.2",
+    "fast-memoize": "^2.5.2",
+    "find-up": "5.0.0",
+    "fp-and-or": "^1.0.2",
+    "hosted-git-info": "^7.0.1",
+    "ini": "^4.1.2",
+    "js-yaml": "^4.1.0",
+    "json-parse-helpfulerror": "^1.0.3",
+    "jsonlines": "^0.1.1",
     "lockfile-lint": "^4.13.2",
+    "lodash": "^4.17.21",
+    "make-fetch-happen": "^13.0.0",
     "markdownlint-cli": "^0.39.0",
     "mocha": "^10.3.0",
     "npm-run-all": "^4.1.5",
+    "p-map": "^4.0.0",
+    "pacote": "17.0.6",
+    "parse-github-url": "^1.0.2",
+    "picomatch": "^4.0.1",
     "prettier": "^3.2.5",
+    "progress": "^2.0.3",
+    "prompts-ncu": "^3.0.0",
+    "rc-config-loader": "^4.1.3",
+    "remote-git-tags": "^3.0.0",
+    "semver": "^7.6.0",
+    "semver-utils": "^1.1.4",
     "should": "^13.2.3",
     "sinon": "^17.0.1",
+    "source-map-support": "^0.5.21",
+    "spawn-please": "^3.0.0",
     "strip-ansi": "^7.1.0",
-    "ts-node": "^10.9.2",
```

Comment:
If ts-node is removed, we should remove/replace `ts-node/register` from the mocha config section (below).